test/bio_base64_test.c

```diff
@@ -241,7 +241,13 @@ static int test_bio_base64_run(test_case *t, int llen, int wscnt)
     if (n1 > 0)
         BIO_write(bio, encoded, n1);
 
-    b64 = BIO_new(BIO_f_base64());
+    if ((b64 = BIO_new(BIO_f_base64())) == NULL) {
+        OPENSSL_free(raw);
```

Comment:
I would clean this test up rather than block copy.... i.e.

Set everything initially at the start
````
    unsigned char *raw = NULL;
    unsigned char *out = NULL;
    BIO *b64 = NULL;
    int ret = -1;
````
and just call
````
goto err;
````
In lots of places.